tenable/io/v3/was/folders.py

```diff
@@ -0,0 +1,94 @@
+'''
+Folders
+========
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`folders <folders>` API.
```

Comment:
@harshshah-tenable @parvdesai-tenable We have folders endpoint in VM as well as WAS. Would we require to do that bifurcation here:
:devportal:`folders <folders>` API?

Reply:
Updated the documentation to point to WAS folders api